src/Symfony/Component/Validator/Tests/Constraints/UniqueTest.php

```diff
@@ -35,6 +36,20 @@ public function testAttributes()
         self::assertSame(['my_group'], $cConstraint->groups);
         self::assertSame('some attached data', $cConstraint->payload);
     }
+
+    public function testInvalidNormalizerThrowsException()
```

Comment:
We should also test that setting the new option via attributes work. You could add something like `normalizer: 'intval'` to one of the attribute blocks.